package.json

```diff
@@ -19,9 +19,14 @@
 	"devDependencies": {
```

Comment:
The whole PR can be made without changing dependencies here, can you rollback these changes & the ones to yarn.lock? 👀